Formula/latino.rb

```diff
@@ -0,0 +1,24 @@
+class Latino < Formula
+  desc "Lenguaje de programación de código abierto para latinos y de habla hispana"
+  homepage "https://www.lenguajelatino.org/"
+  url "https://github.com/lenguaje-latino/Latino/archive/v1.3.0.tar.gz"
+  sha256 "28118a047265394ad9a5fcb2c76895d08af4b43a5886f52afce98a0f8cdc86fe"
+  license "MIT"
+  head "https://github.com/lenguaje-latino/Latino.git"
+  depends_on "cmake" => :build
+  depends_on "make" => :build
+  def install
```

Comment:
Adds some whitespace for readability and removes the dependency on make as discussed earlier

```suggestion
  head "https://github.com/lenguaje-latino/Latino.git"

  depends_on "cmake" => :build

  def install
```